storage/tianmu/core/engine.cpp

```diff
@@ -1953,6 +1950,16 @@ void Engine::AddTableIndex(const std::string &table_path, TABLE *table, [[maybe_
   }
 }
 
+void Engine::DeleteTableIndex(const std::string &table_path, [[maybe_unused]] THD *thd) {
+  std::unique_lock<std::shared_mutex> index_guard(tables_keys_mutex);
+  if (index::TianmuTableIndex::FindIndexTable(table_path)) {
+    index::TianmuTableIndex::DropIndexTable(table_path);
```

Comment:
How to know `DropIndexTable` returns with succeed.  here, we can not know the status of `index::TianmuTableIndex::DropIndexTable`